### DISCUSSION TITLE
在使用Nya LoCyanFrp!时下载frpc时报错

### DISCUSSION THREAD

**Discussion by bsgz** [Muska-Ami/NyaLCF#156] [Q&A] [answered]
如图片这样的报错，即使在GitHub上单独下载frpc放在文件夹里仍然没有用

**Comment by bsgz** [comment]
![QQ截图20240622120321](https://github.com/Muska-Ami/NyaLCF/assets/131172928/940ee57c-1cde-4f74-afb4-52089d5441fa)

**Reply by Muska-Ami** [reply]
服务器爆了

**Reply by bsgz** [reply]
谢谢告知了 ，这么久差点都把这事忘了，所以为什么没有弄个手动下载的功能呢

**Reply by Muska-Ami** [reply, marked as answer]
我们有写通过环境变量来指定 Frpc 的功能，可以通过环境变量方式手动安装，更多请参阅文档

**Reply by bsgz** [reply]
操作文档在哪，我找了很久都没有找到

**Reply by Muska-Ami** [reply]
https://docs-nyalcf.1l1.icu

**Reply by bsgz** [reply]
这段是什么时候加的，以前看好像没有诶

**Reply by Muska-Ami** [reply]
我记得还在用GH Wki的时候就有了，大概是看漏了吧

**Reply by bsgz** [reply]
好吧，谢谢解答了
